clients/go/commands/activateJobs_command.go

```diff
@@ -79,6 +81,12 @@ func (cmd *ActivateJobsCommand) FetchVariables(fetchVariables ...string) Activat
 	return cmd
 }
 
+func (cmd *ActivateJobsCommand) RequestTimeout(timeout time.Duration) ActivateJobsCommandStep3 {
```

Comment:
Why do we need this method if we already have to specify the timeout in NewActivateJobsCommand?

Reply:
zbctl uses this method to configure the timeout.

Reply:
Ah I missed the 2nd commit, my bad